scripts/run_semgrep_ci.py

```
@@ -8,10 +8,10 @@
 def main() -> None:
     """Invoke the Nox semgrep_ci session and exit with its return code."""  # D103
     try:
-        poetry = "poetry"
+        uv = "uv"
         nox = "nox"
         subprocess.run(  # nosec: B603
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)